index.js

```diff
@@ -2,26 +2,21 @@ const path = require('path');
 const notifier = require('node-notifier');
 const NotificationCenter = require('node-notifier/notifiers/notifysend');
 const formatNum = require('format-num')
-const scrapCurrencies = require('./currencyScraper');
 const db = require('./db');
+const getBitsoExchange = require('./services/bitso');
 
 const increaseIcon = 'increase.png';
 const decreaseIcon = 'decrease.png';
 
 const timeToUpdate = 60000;
 const currencies = [
-    'btc_mxn',
```

Comment:
Please put all the currencies in the array again. I'll work on making it more configurable.